lte/gateway/python/magma/subscriberdb/rpc_servicer.py

```diff
@@ -110,22 +101,22 @@ def ListSubscribers(self, request, context):  # pylint:disable=unused-argument
         """
         Returns a list of subscribers from the store
         """
-        try:
-            self._print_grpc(request)
-        except Exception as e:  # pylint: disable=broad-except
-            logging.debug("Exception while trying to log GRPC: %s", e)
+        self._print_grpc(request)
         sids = self._store.list_subscribers()
         sid_msgs = [SIDUtils.to_pb(sid) for sid in sids]
         return subscriberdb_pb2.SubscriberIDSet(sids=sid_msgs)
 
     def _print_grpc(self, message):
```

Comment:
Can we pull these into a single file so we don't have to keep defining it? Other than that LGTM!

Reply:
yep, that is in my backlog. I just don't have the time right now  😭 , but will be done.